test/Broadway/EventSourcing/AbstractEventSourcingRepositoryTest.php

```diff
@@ -25,6 +26,9 @@
 
 abstract class AbstractEventSourcingRepositoryTest extends TestCase
 {
+    /** @var EventBusInterface */
```

Comment:
Can you change this to `TracableEventBus`? The method `->getEvents()` is called and only available on the `TraceableEventBus`. Feel free to change it for the other typehint as well.